src/main/java/com/vention/stock_market_share/service/JwtService.java

```diff
@@ -1,4 +1,4 @@
-/*
+package com.vention.stock_market_share.service;/*
```

Comment:
we don't have security now, we should delete this class

Reply:
Yeah i deleted